app/models/booking.rb

```diff
@@ -0,0 +1,8 @@
+class Booking < ActiveRecord::Base
+  belongs_to :customer, class_name: 'User', foreign_key: :user_id
+  belongs_to :product
+
+  has_one :owner, through: :product
+
+  validates :customer, :product, :check_in, :night, presence: true
```

Comment:
Do not use inline validations (idem for all occurrences).
Sort validations by ascending name (idem for all occurrences).